### DISCUSSION TITLE
v1.14

### DISCUSSION THREAD

**Discussion by tobya** [tobya/DocTo#218] [Announcements]
## Version 1.14

This has been built now on a clean machine so hopefully is clean and straightforward.

ps. I'm not superstitious!

**Full Changelog**: https://github.com/tobya/DocTo/compare/v1.11...v1.14

<hr /><em>This discussion was created from the release <a href='https://github.com/tobya/DocTo/releases/tag/v1.14'>v1.14</a>.</em>

**Comment by supermike** [comment]
windows 10 detected: Trojan:W32/Wacatac.B!ml

**Reply by tobya** [reply]
Thanks.  They are false positives.
